rust/macros/module.rs

```diff
@@ -143,6 +143,7 @@ fn param_ops_path(param_type: &str) -> &'static str {
     }
 }
 
+#[allow(clippy::type_complexity)]
```

Comment:
https://github.com/Rust-for-Linux/linux/pull/898

Reply:
Thanks Alice!